snuba/web/views.py

```diff
@@ -426,9 +420,11 @@ def eventstream(*, dataset: Dataset):
 
         if type_ == "insert":
             from snuba.consumer import ConsumerWorker
+
```

Comment:
intentional ?

Reply:
Oops. I somehow made the exact same change in another PR as well without noticing. Seem to have a habit of putting an empty line after imports.

Reply:
Jk, actually black does that. That makes way more sense.